test/depletion_test.jl

```diff
@@ -9,16 +9,21 @@ T = Float32
     calculate_electric_potential!(sim, refinement_limits=0.01)
     id = SolidStateDetectors.determine_bias_voltage_contact_id(sim.detector)
     calculate_weighting_potential!(sim, id, refinement_limits=0.01)
-    SolidStateDetectors._adapt_weighting_potential_to_electric_potential_grid!(sim, id)
+    SolidStateDetectors._adapt_weighting_potential_to_electric_potential_grid!(
+        sim, id)
     U_est = ustrip(estimate_depletion_voltage(sim)) # around 2600
     ΔU = 50
     # simulate over and under depletion voltage
     U₋ = U_est - ΔU
     U₊ = U_est + ΔU
-    sim.detector = SolidStateDetector(sim.detector, contact_id=1, contact_potential=U₊)
+    sim.detector = SolidStateDetector(
+        sim.detector, contact_id=1, contact_potential=U₊, 
+        depletion_handling=true)
     calculate_electric_potential!(sim, refinement_limits=0.01)
     undepleted = is_depleted(sim.point_types)
```

Comment:
This is still missing an exclamation mark here.
We expect it to be undepleted, so this line should be: !is_depleted